baybe/simulation/scenarios.py

```diff
@@ -136,7 +137,14 @@ def unpack_simulation_results(array: DataArray) -> pd.DataFrame:
     # Simulate and unpack
     result_variable = "simulation_result"
     batch_simulator = make_xyzpy_callable(result_variable)
-    da_results = batch_simulator.run_combos(combos)[result_variable]
+
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            "ignore", module="baybe.recommenders.pure.nonpredictive.base"
+        )
+        da_results = batch_simulator.run_combos(combos)[result_variable]
+    warnings.resetwarnings()
```

Comment:
This resets all warning filters that might have been set or changed by the user. In my opinion, this is too restrictive, and might cause problems. Is it possible to just reset the warning for the one that we actually want to filter?

Reply:
rights, its a leftover not needed at all, removed